package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nextcloud/cypress",
-  "version": "1.0.0",
```

Comment:
Let's release 1.0.0 first? xD

Reply:
Also, maybe as beta.0 ?
```suggestion
  "version": "1.0.0-beta.0",
```

Reply:
Already did but without build files :see_no_evil:

Reply:
Need a new tag now

Reply:
Why is this merged?